src/opm/output/eclipse/EclipseIO.cpp

```diff
@@ -263,7 +263,7 @@ void EclipseIO::writeTimeStep(const SummaryState& st,
     if (!isSubstep) {
         for (const auto& report : schedule.report_config(report_step)) {
             std::stringstream ss;
-            RptIO::write_report(ss, report.first, report.second, schedule, report_step);
+            RptIO::write_report(std::cout, report.first, report.second, schedule, report_step);
```

Comment:
You're of course free to do whatever you need to develop this feature on your local system, but this particular aspect&mdash;unconditional output to `std::cout`&mdash;is not appropriate for Flow's master branch.  As I mentioned in my earlier [comment](https://github.com/OPM/opm-common/pull/1672#issuecomment-612478962), this renders the simulator's command window output unusable for certain field cases and is the reason we implemented the work-around in #1690.

Reply:
Yeah, this will be rebased out as part of moving this PR from draft to review.